libs/crypto/src/mcl_dkg.cpp

```diff
@@ -400,14 +405,12 @@ PrivateKey SignatureAggregationCoefficient(PublicKey const &             notaris
 {
   PrivateKey coefficient;
 
+  std::string concatenated_keys;
+  concatenated_keys.reserve(48 + (cabinet_notarisation_keys.size() + 1) * 310);
```

Comment:
Seems unclear where 48 and 310 are from? Could they be constexpr somewhere? I am not sure I understand the hygenic reuse comment

Reply:
The reserving of the 48 bytes was a figure David gave and the 310 is the size of each public key. I can make these const expressions. The hygenic reuse comment refers to the fact that the hash function (which is also used in the DKG) is being used here for a different purpose and to be completely safe we reserve some bytes at the front so there is no way two of the same strings could be hashed with the same hash function in the different contexts.